Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html

```diff
@@ -1,145 +1,114 @@
+
+Functional object property denotes cardinality of (1) such relationship between 2 instances.
+For example, John (x) is connected to Mary (y) by object-property "has father" is unique.
+That is, Steve (y) can only bear the object-property relationship "has_father" with one other individual, namely, John (x).
+
+Transitive object-property goes according to the following:
+(x) relates to (y) by object-property 1 and (y) related to (z) by same object-property 1, 
+So (x) also relates to (z) by virture of object-property 1.
+x --> y, y --> z Therefore x --> z
+
+When running a reasoner the combination of these becomes undecideable because a functional property must only be only be obtained between one individual and another,
+however, if it is applied transitively it whill carry over to subsequent pairs who are also connected by this same object property.
+
+
+2.) INVERSE-FUNCTIONALITY AND TRANSITIVITY BECOMES UNDECIDEABLE
+
+Inverse functionality provides that each individual paired with another with this relation can have at most one incoming connection of said object property.
+Ex: Inverse_Functional_Property "father of";
+    Father_of (Joey, Richard)      = "Joey is father_of Richard"
+--> This means that the instance, Richard, can only be inverse-fucntionally related to Joey and no other instance because any person can only have one father.
+--> Conversely the object-property "father_of" may be obtained between Joey and other instances, who will only have this relation which Joey. 
+
+Transitive object-property goes according to the following:
+(x) relates to (y) by object-property 1 and (y) related to (z) by same object-property 1, 
+So (x) also relates to (z) by virture of object-property 1.
+x --> y, y --> z Therefore x --> z
+
+When running a reasoner the combination of these becomes undecideable because a functional property must only be only be obtained between one individual and another,
+however, if it is applied transitively it whill carry over to subsequent pairs who are also connected by this same object property.
+
+
+3.) SYMMETRY + A-SYMMETRY = UNSATISFIABLE 
+
+Symmetry declares that if (x) is related to (y) by object_property_1, then (y) is related to (x) by virtue of same object_property_1
+Ex: Symmtric_Object_Property (sister)
+    Object_Property_Assertion (sister, Mary, Jen)  = "Mary is sister of Jen"
+    It follows that "Jen is sister of Mary" because the relation is symmetrically held between both instances. 
+
+Asymmetry provides that if (x) is related to (y) by object_property_3 then (y) CANNOT relate to (x) by object_property_3
+Ex: Asymmetric_Object_Property (Parent_of)
+    Object_Property_Assertion (parent_of, Vivian, Chris)  = "Vivian is a parent of Chris"
+    It holds that Chris CANNOT be a parent of Vivian. 
+
+The joint of these leads to unsatisfiability because an object property cannot relate mutually between instances while also only applying one direction between one to the other.
```

Comment:
Let R be an object property that is both symmetric and asymmetric. Symmetry implies that for any xRy, it must hold that yRx. However, asymmetry requires that if xRy, then it cannot be the case that yRx. These two conditions are logically contradictory, resulting in unsatisfiability